docker/Dockerfile

```diff
@@ -161,39 +158,34 @@ RUN $VENV/bin/jupyter notebook --generate-config; \
     echo "c.NotebookApp.password='sha1:12bff019c253:9daecd92c2e9bdb10b3b8a06767a74a0fe078d7c'">>/home/docker/.jupyter/jupyter_notebook_config.py
 
 WORKDIR /home/docker/
-# COPY ./tvb-multiscale/docker/installation_test.py /home/docker/
-#For running instalation test
-# CMD $VENV/bin/python installation_test.py
 
 # Activate the rights python env
 ENV PYTHONPATH $VENV
 ENV SLI_PATH $VENV
 RUN echo "source $VENV/bin/activate" >> .bashrc
 
-# For running tests
-CMD $VENV/bin/python /home/docker/packages/tvb-multiscale/docker/installation_test.py
+# For running examples
+#CMD $VENV/bin/python /home/docker/packages/tvb-multiscale/docker/installation_test.py
+# CMD $VENV/bin/python /home/docker/packages/tvb-multiscale/tvb_nest/examples/red_ww_exc_io_inh_i.py
+# CMD $VENV/bin/python /home/docker/packages/tvb-multiscale/tvb_nest/examples/rate_ww_ampa_nmda_gaba.py
+
 # Test NEST installation
 # RUN cd $NEST; make installcheck
 # CMD $VENV/bin/python /home/docker/packages/tvb-multiscale/docker/module_build_test.py
-# CMD cd /home/docker/packages/tvb-multiscale; $VENV/bin/pytest tvb_nest/tests
+
 # pynestml installation test:
 # CMD cd /home/docker/packages/nestml; $VENV/bin/pytest setup.py test
+
 # pyNN-NEST installation test:
 # CMD $VENV/bin/python -c "import pyNN.nest as sim; sim.setup(); sim.end(); print(sim)";
 
+#For running tests
+#CMD $VENV/bin/pytest /home/docker/packages/tvb-multiscale/tvb_nest/tests
+
 #For running tests with coverage
-#CMD $VENV/bin/pytest --cov /home/docker/packages/tvb-multiscale/tvb_nest/tests
+#CMD $VENV/bin/pytest --cov=nest_tests_coverage /home/docker/packages/tvb-multiscale/tvb_nest/tests
 
-# CMD $VENV/bin/python /home/docker/packages/tvb-multiscale/tvb_nest/examples/red_ww_exc_io_inh_i.py
-# CMD $VENV/bin/python /home/docker/packages/tvb-multiscale/tvb_nest/examples/rate_ww_ampa_nmda_gaba.py
+#Run jupyter notebooks
 CMD $VENV/bin/jupyter notebook --ip 0.0.0.0 --no-browser --allow-root
```

Comment:
Please leave the CMD to launch Jupyther notebook. It will be the one we advertise to novice users